source/zh-cn/docs/gitlab-pages.md

```diff
@@ -25,19 +28,21 @@ pages:
   only:
     - master
 ```
-5. *username*.gitlab.io should be up and running, once GitLab CI finishes the deployment job,
-6. (Optional) If you wish to inspect the generated site assets (html, css, js, etc), they can be found in the [job artifact](https://docs.gitlab.com/ee/user/project/pipelines/job_artifacts.html).
 
-### Project page
+5. GitLab CI 应该会自动开始运行，构建成功以后你应该可以在 `https://<你的 GitLab 用户名>.gitlab.io` 查看你的网站。
+6. (可选) 如果你需要查看生成的文件，可以在 [job artifact](https://docs.gitlab.com/ee/user/project/pipelines/job_artifacts.html) 中找到。
 
-If you prefer to have a project page on GitLab:
+## Project page
 
-1. Go to `Settings -> General -> Advanced -> Change path`. Change the value to a name, so the website is available at <b>username.gitlab.io/*name*</b>. It can be any name, like *blog* or *hexo*.
-2. Edit **_config.yml**, change the `root:` value from `""` to `"name"`.
-3. Commit and push.
+如果你更希望你的站点部署在 `<你的 GitLab 用户名>.gitlab.io` 的子目录中，你的 repository 需要直接命名为子目录的名字，这样你的站点可以通过 `https://<你的 GitLab 用户名>.gitlab.io/<repository 的名字>` 访问。你需要检查你的 Hexo 配置文件，将 `url` 的值修改为 `https://<你的 GitLab 用户名>.gitlab.io/<repository 的名字>`、将 `root` 的值修改为 `/<repository 的名字>/`
 
+{% note info %}
+请注意，以上教程翻译自 Hexo 文档英文版，仅做参考；译者在翻译过程中发现很多语焉不详的内容。通过搜索引擎可以找到更多教程。
```

Comment:
> 译者在翻译过程中发现很多语焉不详的内容
If there is any unclear instruction, you should've mention it in #1108.

Reply:
I have just copied the same note from `GitHub Pages` guide, as the guide provided by Hexo's documentation are  relatively unclear compared to the "hands-on Hexo teaching" tutorial articles written by Hexo users on their blogs.

It is not neccsary for Hexo document to cover all the use cases and give much more details. That is exactly what we do currently, providing a simple instruction.
But still, we need to inform Hexo users that there are many more use cases and details that is not included in Hexo documents.

Reply:
Please remove that note from `GH Pages` guide as well, it shouldn't be there in the first place.

> we need to inform Hexo users that, there are many more use cases and details are not included in Hexo documents.

We don't need to, the instruction is meant to be generic for most use cases. Advanced users shouldn't rely on that. Besides, the note currently says "unclear instruction", not "doesn't cover all use cases".

Reply:
Removed.